src/convert.jl

```diff
@@ -103,6 +112,11 @@ function trixi2vtk(filename::AbstractString...;
     verbose && println("| Reading mesh file...")
     @timeit "read mesh" mesh = Trixi.load_mesh_serial(meshfile; n_cells_max=0, RealT=Float64)
 
+    # Check compatibility of the mesh type and the output format
+    if format == :vti && !(mesh isa Trixi.TreeMesh)
```

Comment:
```suggestion
    if format == :vti && !(mesh isa Trixi.TreeMesh{2})
```

Reply:
Does this work for serial and parallel meshes?

Reply:
Yes: https://github.com/trixi-framework/Trixi.jl/blob/c8f4cd07d0349d8b40891386e4286165371d226d/src/meshes/tree_mesh.jl#L28